### PR TITLE
Adopt core embed window services

### PR DESCRIPTION
**Pull request by dvoytenko**
This is a simplified solution for core service embedding for friendly iframes. A more complete solution would also adopt services contributed by extensions. However, that will require a deeper refactoring and thus postponed until the need arises.

### CODE REVIEW

File: src/service.js [hunk 1]
```diff
@@ -565,6 +582,44 @@ function disposeServiceInternal(id, service) {
...
+
+
+/**
+ * Asserts that the specified service implements `EmbeddableService` interface
+ * and typecasts the instance to `EmbeddableService`.
+ * @param {!Object} service
+ * @return {!EmbeddableService}
+ */
+export function assertEmbeddable(service) {
+  dev().assert(isEmbeddable(service),
```
Comment (cramforce) [inline src/service.js]: Please manually check that this compiles away.

Comment (dvoytenko) [inline src/service.js]: Yes. It's completely inlined.

Comment (cramforce) [inline src/service.js]: It should be DCEed. Is it?

Comment (dvoytenko) [inline src/service.js]: Yes. I meant to say it directly calls `.adtopEmbedWindow` on the services themselves.

**Pull request merged**